src/components/Calculator.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import './Calculator.css';
+
+class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className="grid-container">
+        <div className="span-4">0</div>
+        <button type="button">AC</button>
+        <button type="button">+/-</button>
+        <button type="button">%</button>
+        <button type="button" className="operation">
+          ÷
+        </button>
+        <button type="button">7</button>
+        <button type="button">8</button>
+        <button type="button">9</button>
+        <button type="button" className="operation">
+          x
+        </button>
+        <button type="button">4</button>
+        <button type="button">5</button>
+        <button type="button">6</button>
+        <button type="button" className="operation">
+          -
+        </button>
+        <button type="button">1</button>
+        <button type="button">2</button>
+        <button type="button">3</button>
+        <button type="button" className="operation">
+          +
+        </button>
+        <button type="button" className="span-2">
+          0
+        </button>
+        <button type="button">.</button>
+        <button type="button">=</button>
```

Comment:
Please make sure that the equal (=) sign operator background is also the same color `#f49040` that you used for your operators as in the design specifications.
![image](https://user-images.githubusercontent.com/38088818/170049785-927a56fd-52c8-4782-887c-255c596e983d.png)

Required design:
![image](https://user-images.githubusercontent.com/38088818/170050046-3cfb1121-74c7-423f-aef2-f4b84f16323b.png)

Reply:
@KDlamini  Done, Thank you I didn't notice that 😞